ComposeSample/app/src/androidTest/java/com/microsoft/device/display/samples/composesample/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.composesample
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
```

Comment:
If you don't plan to add now UI tests, you can get rid of this class as is just noise that has to be maintained. Thanks!